membership-attribute-service/app/services/AttributesMaker.scala

```diff
@@ -40,12 +41,13 @@ class AttributesMaker extends LazyLogging {
       val recurringContributionPaymentPlan: Option[String] = contributionSub.flatMap(getTopPlanName)
       val membershipJoinDate: Option[LocalDate] = membershipSub.map(_.startDate)
       val latestDigitalPackExpiryDate: Option[LocalDate] = Some(subsWhichIncludeDigitalPack.map(_.termEndDate)).filter(_.nonEmpty).map(_.max)
+      val staffExpiryDate: Option[LocalDate] = tier.exists(_.equalsIgnoreCase("staff")).option(now.plusDays(1))
```

Comment:
so I guess we're telling it to call back in every day to check the expiry, I'm not expecting that to overload the servers though!

Reply:
does tier really come in mixed case? I thought it was the product name which is "Staff"

Reply:
That expiry date is not actually cacheable anyway because the product can be cancelled and refunded before its term or paid-in-advance period ends.